app/helpers/template_helpers.py

```diff
@@ -327,11 +328,47 @@ class CensusNISRASurveyConfig(
     )
 
 
+@dataclass
+class BusinessSurveyConfig(
+    SurveyConfig,
+):
+    base_url: str = BUSINESS_BASE_URL
+    account_service_url: str = f"{BUSINESS_BASE_URL}/surveys/todo"
+
+    footer_links: Iterable[MutableMapping] = field(
+        default_factory=lambda: [
+            Link(
+                lazy_gettext("What we do"), f"{DEFAULT_EN_BASE_URL}/aboutus/whatwedo/"
+            ).__dict__,
+            Link(
+                lazy_gettext("Contact us"), f"{BUSINESS_BASE_URL}/contact-us/"
+            ).__dict__,
+            Link(
+                lazy_gettext("Accessibility"),
+                f"{DEFAULT_EN_BASE_URL}/help/accessibility/",
+            ).__dict__,
+        ],
+        compare=False,
+        hash=False,
+    )
+
+    footer_legal_links: Iterable[Mapping] = field(
+        default_factory=lambda: [
+            Link(lazy_gettext("Cookies"), f"{BUSINESS_BASE_URL}/cookies/").__dict__,
+            Link(
+                lazy_gettext("Privacy and data protection"),
+                f"{BUSINESS_BASE_URL}/privacy-and-data-protection/",
+            ).__dict__,
+        ],
+        compare=False,
+    )
```

Comment:
I think the links should be placeholders since we don't have confirmed locations yet. Some of the content from RAS might move also. If we decided to keep it, the base URL need to be updated to `"https:www.//ons.gov.uk"`.

Reply:
Problem with half baked stories, all our test schemas would use this and links would be broken/placeholders/might be wrong. I am more incline to remove the links completely and not have loose ends, which might be confused as already being implemented

Reply:
Changed to # :frowning: